packages/config/config/devices/0x000c/hs-ps100.json

```diff
@@ -0,0 +1,114 @@
+{
+	"manufacturer": "HomeSeer Technologies",
+	"manufacturerId": "0x000c",
+	"label": "HS-PS100",
+	"description": "Z-Wave Presence Sensor",
+	"devices": [
+		{
+			"productType": "0x0204",
+			"productId": "0x0002"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Basic Set",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Off Delay After Motion",
+			"valueSize": 4,
+			"unit": "seconds",
+			"minValue": 10,
+			"maxValue": 3600,
+			"defaultValue": 10
+		},
+		{
+			"#": "2",
+			"label": "Distance Report Interval",
+			"valueSize": 4,
+			"unit": "seconds",
+			"minValue": 2,
+			"maxValue": 600,
+			"defaultValue": 10
+		},
+		{
+			"#": "3",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Bluetooth",
+			"description": "After changing this setting, the unit will need to be power cycled before it will take effect."
+		},
+		{
+			"#": "4",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Motion LED",
+			"description": "Enable / Disable the LED flash when motion is sensed (red) and times out (green).",
+			"defaultValue": 1
+		},
```

Comment:
Thanks! I changed the labels for these two, because we avoid putting "Enable/Disable" in labels of parameters where the only choice is "Enable" or "Disable"